doc/1/plugins/guides/events/core-hotelclerk-addSubscription/index.md

```diff
@@ -0,0 +1,29 @@
+---
+code: true
+type: page
+title: core:hotelClerk:addSubscription
+---
+
+# core:hotelClerk:addSubscription
+
+
+| Arguments  | Type              | Description                           |
+| ---------- | ----------------- | ------------------------------------- |
+| `diff`     | <pre>object</pre> | Contains information about subscription |
+
+Triggered whenever a [subscription](/core/1/api/controllers/realtime/subscribe) is added.
+
+## diff
+
+The provided `diff` object has the following properties:
+
+| Properties     | Type                 | Description                                                                                                       |
+| -------------- | -------------------- | ----------------------------------------------------------------------------------------------------------------- |
+| `roomId`       | <pre>integer</pre>   | New room unique identifier                                                                                    |
```

Comment:
The room may not be "new" if a subscription is added to an already existing room.

And room identifiers are strings:

```suggestion
| `roomId`       | <pre>string</pre>   | Room unique identifier                                                                                    |
```